compiler/rustc_smir/src/rustc_internal/mod.rs

```diff
@@ -0,0 +1,12 @@
+//! Module that implements the bridge between Stable MIR and internal compiler MIR.
+//!
+//! For that, we define APIs that will temporarily be public to 3P that exposes rustc internal APIs
+//! until stable MIR is complete.
+
+use crate::stable_mir::CrateItem;
+
+pub type DefId = rustc_span::def_id::DefId;
```

Comment:
Shouldn't this be wrapped in a newtype to avoid leaking implementation details?

Reply:
This module is specifically be forever unstable so that we allow nightly users to do more than what stable-mir allows, while still mostly using the stable-mir API